orchest-sdk/python/orchest/utils.py

```diff
@@ -8,6 +8,22 @@
 from orchest.pipeline import Pipeline
 
 
+def pretty_print_env_var_suggestion(pipeline: Pipeline):
```

Comment:
```suggestion
def pretty_print_env_var_suggestion(pipeline: Pipeline) -> None:
```

Common practice to explicitly return `None` if applicable.